# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/frontend/Stencil.h
# Commit: ca60758cd534
# Full Hash: ca60758cd534c0f3bb04a7acda15eb41a67e7389
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2020-11-14 09:46:25
# Regressor Bug: 1675241
# File Overlap Count: 1
# Description:
#   Bug 1675241 - Part 12: Use TaggedParserAtomIndex in ObjLiteralStencil. r=tcampbell
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96919
# ==============================================================================

diff -r 8af30c6d9bb9 -r ca60758cd534 js/src/frontend/Stencil.h
--- a/js/src/frontend/Stencil.h	Sat Nov 14 00:03:36 2020 +0000
+++ b/js/src/frontend/Stencil.h	Sat Nov 14 00:03:45 2020 +0000
@@ -527,6 +527,12 @@
 #endif
 };
 
+#if defined(DEBUG) || defined(JS_JITSPEW)
+void DumpTaggedParserAtomIndex(js::JSONPrinter& json,
+                               TaggedParserAtomIndex taggedIndex,
+                               CompilationStencil* compilationStencil);
+#endif
+
 } /* namespace frontend */
 } /* namespace js */
 